packages/apollos-data-prayer/src/prayer-requests/__tests__/resolver.test.js

```diff
@@ -129,119 +131,13 @@ describe('PrayerRequest resolvers', () => {
         campusPrayers(campusId: "Campus:4f68015ba18662a7409d1219a4ce013e") {
           id
           firstName
-          lastName
           text
-          requestedByPersonAliasId
-          campus {
```

Comment:
I don't necessarily mind not returning everything that could be returned. But I kind of feel like we should be returning campus on the query that is pulling requests by campus.

Reply:
yea, I'll just make the mock data be from the same campus so it looks right